src/app/export-execution/page.tsx

```diff
@@ -0,0 +1,31 @@
+'use client';
+
+import { Anchor, Flex, Pill, Stack, Title } from '@mantine/core';
+import { useSearchParams } from 'next/navigation';
+
+export default function ExecutionPage() {
+  const searchParams = useSearchParams();
+
+  const encodedContentLocation = searchParams.get('contentLocation');
+  if (!encodedContentLocation) return <Title>Error: Bad content-location</Title>;
+  const contentLocation = decodeURIComponent(encodedContentLocation);
+
+  return (
+    <Stack>
+      <Flex>
+        <Title order={3} mr="lg">
+          Status:
+        </Title>
+        <Pill size="xl" bg="green" c="white" fw={700}>
+          202
```

Comment:
This is a fake result in this case but we will only get to this page if we had a 202 successful kickoff.